### PR TITLE
Unit test: Add test for all transaction types  in Runtime executeTx with SDK txn format.

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ from `algosdk` and sends it to the network.
 - Teal v6 support:
   - Add `Txn LastLog` opcode.
   - Add `Txn StateProofPK` opcode.
+- Add unit tests for all transaction types in runtime executeTx.
```
Comment (thdailong) [inline CHANGELOG.md:89]: You place this line wrong position. It should be in `Unreleased` header

File: packages/runtime/test/integration/execute-transaction.ts [hunk 2]
```diff
@@ -226,4 +251,264 @@ describe("Algorand Smart Contracts - Execute transaction", function () {
...
+					payFlags: { totalFee: 1000 },
+				},
+			];
+
+			runtime.executeTx(tx);
+		}
+
+		syncAccounts();
+		assert.equal(alice.balance(), initialBalance - 2000n);
+	});
```
Comment (thdailong) [inline packages/runtime/test/integration/execute-transaction.ts:285]: You should also check whether the account have opt-in the application or not

File: packages/runtime/test/integration/execute-transaction.ts [hunk 3]
```diff
@@ -226,4 +251,264 @@ describe("Algorand Smart Contracts - Execute transaction", function () {
...
+			fromAccount: john.account,
+			assetID: assetId,
+			payFlags: { totalFee: 1000 },
+		};
+
+		runtime.executeTx([destroyParam]);
+
+		syncAccounts();
+		assert.equal(john.balance(), initialBalance - 1000n);
+	});
```
Comment (thdailong) [inline packages/runtime/test/integration/execute-transaction.ts:413]: I recommend you should add check whether the asset is still exist or not

File: CHANGELOG.md (hunk 1 above)
Comment (rmeena840) [reply CHANGELOG.md]: Done

File: packages/runtime/test/integration/execute-transaction.ts (hunk 2 above)
Comment (rmeena840) [reply packages/runtime/test/integration/execute-transaction.ts:285]: ok. done

File: packages/runtime/test/integration/execute-transaction.ts (hunk 3 above)
Comment (rmeena840) [reply packages/runtime/test/integration/execute-transaction.ts:413]: ok. done

**Comment by rmeena840** [comment]
@thdailong Ready for another review.

File: packages/runtime/test/integration/execute-transaction.ts [hunk 4]
```diff
@@ -224,6 +246,284 @@ describe("Algorand Smart Contracts - Execute transaction", function () {
 			];
 
 			runtime.executeTx(tx);
+			syncAccounts();
+			// verify
+			assert.exists(alice.getAppFromLocal(appInfo?.appID))
+		}
+	});
+
+	it("Should opt-in and call app, through execute transaction", () => {
```
Comment (thdailong) [inline packages/runtime/test/integration/execute-transaction.ts:255]: Line of codes in 231 and 255 are pretty much the same, is this one duplicated?

File: packages/runtime/test/integration/execute-transaction.ts (hunk 2 above)
Comment (rmeena840) [reply packages/runtime/test/integration/execute-transaction.ts]: Duplicated by mistake. Removed it.

File: packages/runtime/test/integration/execute-transaction.ts (hunk 4 above)
Comment (rmeena840) [reply packages/runtime/test/integration/execute-transaction.ts]: Duplicated by mistake. Removed it.

**Review by thdailong — APPROVED** [review]
This look better now

**Pull request merged**